moveit_setup_assistant/src/widgets/planning_groups_widget.cpp

```diff
@@ -102,10 +102,12 @@ PlanningGroupsWidget::PlanningGroupsWidget(QWidget* parent, moveit_setup_assista
   QVBoxLayout* layout = new QVBoxLayout();
 
   // Top Label Area ------------------------------------------------
-  HeaderWidget* header =
-      new HeaderWidget("Planning Groups", "Create and edit planning groups for your robot based on joint collections, "
-                                          "link collections, kinematic chains and subgroups.",
-                       this);
+  HeaderWidget* header = new HeaderWidget(
+      "Planning Groups", "Create and edit planning groups for your robot based on joint collections, "
+                         "link collections, kinematic chains and subgroups.\n"
+                         "A planning group defines the set of (joint, link) pairs considered for planning. "
```

Comment:
I am not sure what you mean by that statement.
So I'm pretty sure this is not very helpful to end-users...

What about

    Planning groups define the kinematic structures MoveIt can reason about.

or something similar instead?

Reply:
The planning group is not related to a kinematic _structure_ (which is a chain or a tree).

I want to clarify, that a planning group defines the set of joints that are actually used for planning with this group, i.e. joints that are sampled / moved.
I'm not sure, whether the planning group also restricts collision checking.

Reply:
Agreed, "kinematic structure" is not quite correct either (although in most practical cases it is).

JMGs contain a set of joints and a set of links, yes.
What I believe to be confusing in your explanation is the "(joint,link) pairs" bit.

> I'm not sure, whether the planning group also restricts collision checking.

Yes it does.

Reply:
> What I believe to be confusing in your explanation is the "(joint,link) pairs" bit

But that's the crucial point. If you add a link to the group, its parent joint is added too and vice versa. Hence, its all about (joint,link) pairs. If you manage to pack this into a single sentence, you are very welcome to do so.
Probably, we should go for another comment along this line in the joints / links widgets.

Reply:
> If you add a link to the group, its parent joint is added too and vice versa

This is definitely not described in the assistant at the moment and I was not aware of this behavior. In this case I agree with the formulation.

>  Probably, we should go for another comment along this line in the joints / links widgets.

Yes. Could you add that please?

Reply:
Done. However, it wasn't possible to add this phrase in the sub widgets. It's part of the header now as well.